src/components/LessonActions/components/LessonEdit/index.js

```diff
@@ -1,31 +1,36 @@
 import React from 'react'
 import {Button} from 'egghead-ui'
-import {
-  uploadVideoTitleText,
-  editLessonTitleText,
-  replaceVideoTitleText,
-} from 'utils/text'
+import {Text} from 'react-localize'
 import Anchor from 'components/Anchor'
 
 export default ({lesson}) => (
   <div className='flex'>
     {
       lesson.edit_lesson_http_url
-        ? <div className='pa1'>
+        ? <div className='pr2 pb2'>
             <Anchor url={lesson.edit_lesson_http_url}>
-              <Button size='extra-small'>
-                {editLessonTitleText}
+              <Button 
+                color='base'
+                size='extra-small'
+              >
+                <Text message='lessonEdit.edit' />
               </Button>
             </Anchor>
           </div>
         : null
     }
     {
       lesson.upload_lesson_http_url 
-        ? <div className='pa1'>
+        ? <div className='pb2'>
             <Anchor url={lesson.upload_lesson_http_url}>
-              <Button size='extra-small'>
-                {`${lesson.wistia_id ? replaceVideoTitleText : uploadVideoTitleText} Video`}
+              <Button
+                color='base'
+                size='extra-small'
+              >
+                {lesson.wistia_id
+                  ? <Text message='lessonEdit.replaceVideo' />
+                  : <Text message='lessonEdit.uploadVideo' />
+                }
```

Comment:
Fixed button padding so left side is flush with other elements
<img width="253" alt="button-padding" src="https://cloud.githubusercontent.com/assets/5497885/24312558/c60b57bc-109e-11e7-8624-cdd587cc68b2.png">